psi-mi.owl

```diff
@@ -21,7 +21,7 @@
         <rdfs:comment>The PSI MI schema defines short labels for controlled vocabulary terms</rdfs:comment>
         <rdfs:comment>The correct use of these vocabularies in the PSI Molecular Interaction XML schema is</rdfs:comment>
         <rdfs:comment>The last accession number used in this file is stored in a separate file,</rdfs:comment>
-        <rdfs:comment xml:lang="ebi.ac.uk">The maintenance of this file is ensured by Luana Licata luana.licata@uniroma2.it and Sandra Orchard orchard</rdfs:comment>
+        <rdfs:comment xml:lang="ebi.ac.uk">The maintenance of this file is ensured by Luana Licata luana.licata@uniroma2.it and Sandra Orchard orchard@ebi.ac.uk</rdfs:comment>
```

Comment:
@jmedinaebi I checked on issue #461 and saw you made this pull request. Can you also please remove the invalid language tag here? (`xml:lang="ebi.ac.uk"`) This causes RDF parsing errors. Thank you!

Reply:
I've removed the language tags in this file, there were 9 of them.